stdlib/3/subprocess.pyi

```diff
@@ -4,21 +4,61 @@
 
 from typing import Sequence, Any, Mapping, Callable, Tuple, IO, Optional, Union
 
-# TODO force keyword arguments
-# TODO more keyword arguments
-def call(args: Union[str, Sequence[str]], *, stdin: Any = ..., stdout: Any = ...,
-         stderr: Any = ..., shell: bool = ...,
+# Same args as Popen.__init__
+def call(args: Union[str, Sequence[str]],
```

Comment:
This new definition drops the fact that all arguments except for args are keyword-only.
(That's the right thing to do for 2.7/, but I'm guessing that for 3/ the '*' was actually justified)